CMakeLists.txt

```diff
@@ -7,6 +7,11 @@ cmake_minimum_required(VERSION 2.8)
 
 project(OpenTomb)
 
+if(NOT CMAKE_BUILD_TYPE)
+    set(CMAKE_BUILD_TYPE Debug CACHE STRING "Build type" FORCE)
+    set_property(CACHE CMAKE_BUILD_TYPE PROPERTY STRINGS Debug Release MinSizeRel RelWithDebInfo)
+endif()
+
 if(APPLE)
     file(GLOB OPENAL_SRC
         3rdparty/al/*.c
```

Comment:
But I should repeat again: it's a temporary measure, just to be able to build the documentation. In a few days we should find a solution for a working building system. The previous one was broken and created too much controversy. That doesn't mean that this PR is still valid, though.